usr/sbin/pkcsstats/pkcsstats.mk

```diff
@@ -0,0 +1,16 @@
+sbin_PROGRAMS += usr/sbin/pkcsstats/pkcsstats
+
+usr_sbin_pkcsstats_pkcsstats_LDFLAGS = -lcrypto -ldl -lrt
+
+usr_sbin_pkcsstats_pkcsstats_CFLAGS  =			\
+	-DOCK_TOOL					\
+	-DSTDLL_NAME=\"pkcsstats\"			\
+	-I${srcdir}/usr/include 			\
+	-I${srcdir}/usr/lib/common 			\
+	-I${srcdir}/usr/lib/api
```

Comment:
You also need `-I${top_builddir}/usr/lib/api` since this is where `mechtable-gen.h` is placed.

Reply:
good catch!